### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by jermyk**

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Fixes #
+
+## Description
+
+- 
+-
+
+@neontribe/contemplating-action-dev
```
Comment (katjam) [inline PULL_REQUEST_TEMPLATE.md]: I'll set this up as a team.

Comment (katjam) [reply PULL_REQUEST_TEMPLATE.md]: I've just shortened it to contemplating-action without the dev. Can you please change here too?

**Review by katjam — CHANGES_REQUESTED** [review]
remove the `-dev` part of the team name

**Review by jermyk — COMMENTED** [review]
@katjam Done

**Review by katjam — APPROVED** [review]
Yay. 😎

**Pull request merged**